kyo-core/shared/src/main/scala/kyo/stats/internal/TraceReceiver.scala

```diff
@@ -37,6 +28,15 @@ object TraceReceiver:
             )(using Frame) =
                 Span.noop
 
+    val get: TraceReceiver =
```

Comment:
I was facing compiler crashes in Native. The issue seems the combination of `lazy val` with `ServiceLoader`. The Scala Native developers will investigate. As a workaround, I've removed `lazy`. It's not ideal because failure to load the service instances will result in the class itself failing to load but it seems reasonable since it won't be a silent failure.